churchslavonic.tex

```diff
@@ -1,14 +1,17 @@
+\usepackage{fontspec,metalogo}
+
+\newfontfamily\englishfont[Script=Latin,Ligatures=TeX,HyphenChar=-]{Linux Libertine O}
 \newfontfamily\russianfont[Script=Cyrillic,Ligatures=TeX]{Linux Libertine O}
 \newfontfamily\russianfonttt[Ligatures=TeX]{lmmono10-regular.otf}
 \newfontfamily\russianfontsf[Ligatures=TeX]{lmsans10-regular.otf}
 \newfontfamily\churchslavonicfont[Script=Cyrillic,Ligatures=TeX,HyphenChar=_]{PonomarUnicode.otf}
 \newfontfamily\musicFont[Scale=1.5]{MezenetsUnicode}
 
+\usepackage{luacolor}
```

Comment:
Other PR automatically loads `luacolor` when LuaTeX engine is detected - no need for explicit load.

Reply:
Fixed.